.ci/maven-prepare.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+sudo apt-get install xmlstarlet
+CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
+                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
+echo CURRENT_VERSION="$CURRENT_VERSION"
+
+if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
+  echo "[ERROR] Target Version and Current Version doesn't match."
+  exit 1;
+fi
+
+git config --global user.name 'github-actions[bot]'
```

Comment:
Please make check if such value is not set.

Such shell should be able to be executed by maintainer on his local.
So such command will be damaging for him.

Reply:
Changes Done